### ISSUE TITLE
Enquirer is mind-blowing

### ISSUE THREAD

**Issue by faressoft** [enquirer/enquirer#269]
I am here just to say Enquirer is mind-blowing!! Snippet and Form prompts!!!! Wow

![wbcMnfHqOJX9K](https://user-images.githubusercontent.com/1908658/77044343-88063680-69bf-11ea-873e-f63d137ff5b9.gif)

I am sorry to open an issue for that :p Please close the issue when you see it.

**Comment by jonschlinkert** [comment]
Thanks! Your kind words are very much appreciated!

**Comment by SiddharthShyniben** [comment]
Totally agree! It's so easy to use enquirer instead of calling `readline.createInterface` blah blah blah...
It's good that these kind of libraries exist. They are not core functionality which should totally be written by ourselves but are utilities which should be builtin. They save a lot of time too.

**Comment by tfrancois** [comment]
I totally agree with the original poster.  WOW!  I stumbled on this library and I am stunned by its beauty, power and simplicity.  It is far and away the best of class.  Excellent work and thank you for such an exemplary and extremely useful utility!

Edit:
Having said that - any way to look into my feature request: #477 - thanks in advance!
